### PR TITLE
splitting out tests + adding proper interpolation for stalled cumulatives

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: api/README.V1.md [hunk 1]
```diff
@@ -235,6 +235,23 @@ Utilization of hospital resources.
...
 
+### Metrics
+Calculated metrics data based on known actuals.
+
+| name           | type   | description                                                                       |
+|----------------|--------|-----------------------------------------------------------------------------------|
+| testPositivity | number | Ratio of people who test positive calculated using a 7 day rolling average.       |
+| caseDensity    | number | The number of cases per 100k population calculated using a 7 day rolling average. |
+
+### MetricsTimeseriesRow
```
Comment (benholtzmansmith) [inline api/README.V1.md:246]: Minor: The fact that both of these classes essentially have the same documentation kind of indicates to me that we probably don't need both.

File: api/schemas/AggregateRegionSummary.json [hunk 2]
```diff
@@ -174,6 +174,27 @@
         "ICUBeds"
       ]
     },
+    "Metrics": {
```
Comment (benholtzmansmith) [inline api/schemas/AggregateRegionSummary.json:177]: How do the API schemas get used?

File: api/README.V1.md (hunk 1 above)
Comment (ghop02) [reply api/README.V1.md:246]: Yeah I go back and forth there.  I think the cost of having it is pretty small, but definitely a bit awkward

File: api/schemas/AggregateRegionSummary.json (hunk 2 above)
Comment (ghop02) [reply api/schemas/AggregateRegionSummary.json:177]: They're now used a couple ways.

1. The frontend code actually consumes them to generate typed classes.
2. we use it now to autogenerate the readme documentation

**Pull request merged**